src/zrc/api/schema.py

```diff
@@ -1,6 +1,9 @@
 from django.conf import settings
 
 from drf_yasg import openapi
+from vng_api_common.utils import notification_documentation
+
+from zrc.api.viewsets import ZaakViewSet
```

Comment:
this is an unused import, right?

Reply:
yes, but I needed some way to trigger [this code](https://github.com/VNG-Realisatie/vng-api-common/blob/master/vng_api_common/notifications/viewsets.py#L36) from vng-api-common, because if I don't, the resources and actions for the channel won't be fetched :(.

Reply:
Typical construction for that is to tap into django's [`AppConfig`](https://docs.djangoproject.com/en/2.2/ref/applications/#methods) and perform an import that gives the desired effect during the Django startup/setup phase.